setup.cfg

```diff
@@ -4,8 +4,8 @@ description = A language to describe, manipulate and convert particle decays
 long_description = file: README.md
 long_description_content_type = text/markdown
 url = https://github.com/scikit-hep/decaylanguage
-author = Henry Fredrick Schreiner III, Eduardo Rodrigues
-author_email = henry.schreiner@cern.ch, eduardo.rodrigues@cern.ch
+author = Eduardo Rodrigues, Henry Fredrick Schreiner III
```

Comment:
I changed this to be in a more natural order, also because most of the package is actually the stuff I put in by now. I hope this is OK with you.

Reply:
That's fine. :)